Content.Server/StationEvents/Events/BureaucraticErrorRule.cs

```diff
@@ -29,6 +29,21 @@ protected override void Started(EntityUid uid, BureaucraticErrorRuleComponent co
         if (jobList.Count == 0)
             return;
 
+        // Start of Harmony change of Bureaucratic Error event
+        // - Changed for the sake of people generally not liking when the job they want to play loses all slots within 10 minutes.
+        // - Removed the ability for this event to reduce slots from a job, and replaced with picking a random number of jobs to set to unlimited
+        var num = RobustRandom.Next(component.MinimumJobs, component.MaximumJobs); // Get the number of jobs to become unlimited
+
+        for (var i = 0; i < num; i++)
+        {
+            var chosenJob = RobustRandom.PickAndTake(jobList);
+            if (_stationJobs.IsJobUnlimited(chosenStation.Value, chosenJob))
+                continue;
+
+            _stationJobs.MakeJobUnlimited(chosenStation.Value, chosenJob);
+        }
```

Comment:
This change is missing an end tag
```suggestion
        // Start of Harmony change of Bureaucratic Error event
        // - Changed for the sake of people generally not liking when the job they want to play loses all slots within 10 minutes.
        // - Removed the ability for this event to reduce slots from a job, and replaced with picking a random number of jobs to set to unlimited
        var num = RobustRandom.Next(component.MinimumJobs, component.MaximumJobs); // Get the number of jobs to become unlimited

        for (var i = 0; i < num; i++)
        {
            var chosenJob = RobustRandom.PickAndTake(jobList);
            if (_stationJobs.IsJobUnlimited(chosenStation.Value, chosenJob))
                continue;

            _stationJobs.MakeJobUnlimited(chosenStation.Value, chosenJob);
        }
        // End of Harmony change of Bureaucratic Error event
```